runtime/mux.go

```diff
@@ -55,6 +58,19 @@ func WithForwardResponseOption(forwardResponseOption func(context.Context, http.
 	}
 }
 
+// A QueryParameterParser populates message form HTTP query parameters values
+type QueryParameterParser func(msg proto.Message, values url.Values, filter *utilities.DoubleArray) error
+
+// WithQueryParametersPopulator returns a ServeMuxOption representing new populator used to populate message from query parameters
+//
+// queryParametersPopulator is an option that will be called on the proto.Message
+// url.Values, and utilities.DoubleArray on incoming requests.
+func WithQueryParametersPopulator(queryParametersPopulator QueryParameterParser) ServeMuxOption {
```

Comment:
Lets name this more consistently:
```suggestion
func WithQueryParameterParser(queryParameterParser QueryParameterParser) ServeMuxOption {
```